beta/src/pages/learn/render-and-commit.md

```diff
@@ -1,44 +1,44 @@
 ---
-title: Render and Commit
+title: रेंडर और कमिट
 ---
 
 <Intro>
 
-Before your components are displayed on screen, they must be rendered by React. Understanding the steps in this process will help you think about how your code executes and explain its behavior.
+आपके कंपोनेंट्स स्क्रीन पर प्रदर्शित होने से पहले, उन्हें React द्वारा रेंडर किया जाना चाहिए। इस प्रक्रिया के चरणों को समझने से आपको यह सोचने में मदद मिलेगी कि आपका कोड कैसे निष्पादित होता है और इसके व्यवहार की व्याख्या करता है।
 
 </Intro>
 
 <YouWillLearn>
 
-* What rendering means in React
-* When and why React renders a component
-* The steps involved in displaying a component on screen
-* Why rendering does not always produce a DOM update
+* React में रेंडरिंग का क्याअर्थ है
+* React कब और क्यों एक कॉम्पोनेन्ट रेंडर करता है
+* स्क्रीन पर एक कॉम्पोनेन्ट प्रदर्शित करने में शामिल कदम
+* क्यों रेंडरिंग हमेशा एक डोम अद्यतन उत्पन्न नहीं करता है
 
 </YouWillLearn>
 
-Imagine that your components are cooks in the kitchen, assembling tasty dishes from ingredients. In this scenario, React is the waiter who puts in requests from customers and brings them their orders. This process of requesting and serving UI has three steps:
+कल्पना कीजिए कि आपके कंपोनेंट्स रसोई में रसोइया हैं, सामग्री से स्वादिष्ट व्यंजन इकट्ठा करते हैं। इस परिदृश्य में, React वेटर है जो ग्राहकों से अनुरोध करता है और उन्हें उनके आदेश लाता है। UI का अनुरोध करने और उसे प्रस्तुत करने की इस प्रक्रिया के तीन चरण हैं
 
-1. **Triggering** a render (delivering the guest's order to the kitchen)
-2. **Rendering** the component (getting the order from the kitchen)
-3. **Committing** to the DOM (placing the order on the table)
+1. **ट्रिग्गरिंग** रेंडर (रसोईघर में अतिथि का आदेश पहुंचाना)
+2. **रेंडरिंग** कॉम्पोनेन्ट (रसोई से आदेश प्राप्त करना)
+3. **कमिटिंग** डोम पर (टेबल पर ऑर्डर देकर)
 
 <IllustrationBlock sequential>
-  <Illustration caption="Trigger" alt="React as a server in a restaurant, fetching orders from the users and delivering them to the Component Kitchen." src="/images/docs/illustrations/i_render-and-commit1.png" />
-  <Illustration caption="Render" alt="The Card Chef gives React a fresh Card component." src="/images/docs/illustrations/i_render-and-commit2.png" />
-  <Illustration caption="Commit" alt="React delivers the Card to the user at their table." src="/images/docs/illustrations/i_render-and-commit3.png" />
+  <Illustration caption="ट्रिगर" alt="React as a server in a restaurant, fetching orders from the users and delivering them to the Component Kitchen." src="/images/docs/illustrations/i_render-and-commit1.png" />
+  <Illustration caption="रेंडर" alt="The Card Chef gives React a fresh Card component." src="/images/docs/illustrations/i_render-and-commit2.png" />
+  <Illustration caption="कमिट" alt="React delivers the Card to the user at their table." src="/images/docs/illustrations/i_render-and-commit3.png" />
```

Comment:
`कमिट` => `Commit`

Reply:
Shall I also change the captions of the second illustration starting at line 75? to the English ones

Reply:
@Rohan-Makwana 

Yes. anything which is part of code, we don't want to translate those